CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## 3.0.0 (unreleased)
 
-* Remove pessimistic constraint against future Rails releases.
+* Remove ActiveJob integration. For compatibility reasons, use: [hyperoslo/cellular-activejob][TODO].
```

Comment:
This change will be in a branch for now, but I can create the repo before merging.